packages/fxa-content-server/README.md

```diff
@@ -39,19 +39,22 @@ Any changes made to the css resources, scripts or the template files will automa
 
 ## Testing
 
-### Prerequisites
+### Functional Tests
 
-- Java JDK or JRE (http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+📖 A much more thorough breakdown of Content Server functional tests can be found in our [Ecosystem Platform docs](https://github.com/mozilla/ecosystem-platform/blob/master/docs/fxa-engineering/functional-testing.md).
```

Comment:
Minor thing, probably better to steer people to the github pages URL https://mozilla.github.io/ecosystem-platform/docs/fxa-engineering/functional-testing

Reply:
Great point!